src/slack/tests/main_test.bal

```diff
@@ -67,7 +67,7 @@ function testPostTextMessage() {
             test:assertFail(msg = response.toString());
         }
     } else {
-        test:assertFail(msg = <string>response.detail()?.message);
+        test:assertFail(msg = <string>response.detail()["message"]);
```

Comment:
```suggestion
        test:assertFail(msg = <string>response.message());
```

Reply:
```suggestion
        test:assertFail(msg = response.message());
```
Shall we remove `<string>` part as well?